lib/skate/settings/user.ex

```diff
@@ -87,14 +79,11 @@ defmodule Skate.Settings.User do
     end
   end
 
+  @spec user_ids_for_route_ids(any) :: any
```

Comment:
**question:** is it possible to give this a more specific typing?

Reply:
Good point! Added in [b16ab90](https://github.com/mbta/skate/pull/1798/commits/b16ab90166e2370dfcda1538668577de7c337bbf)